### DISCUSSION TITLE
141947

### DISCUSSION THREAD

**Discussion by Alexjunior9524** [community/community#143226] [Programming Help]
### Body

CHANGELOG.md

### Guidelines

- [X] I have read and understood this category's [guidelines](https://github.com/orgs/community/discussions/51384) before making this post.

**Comment by ebndev** [comment]
Hi @Alexjunior9524, thanks for being a part of the GitHub Community!

In our [Community Code of Conduct](https://github.com/community/community/blob/main/CODE_OF_CONDUCT.md), we ask that members contribute in a positive and constructive way. This type of post is considered "off topic" as GitHub Community Discussions focuses primarily on topics related to GitHub itself or collaboration on project development and ideas. Any repeat and future violations may result in a temporary or indefinite block from the Community. 

We appreciate your understanding.
